main:
  planner_service_name: /path_planning/plan_kinematic_path
  trajectory_filter_service_name: /trajectory_filter_server/filter_trajectory_with_constraints
  segmentation_service_name: /tabletop_segmentation
  grasp_planning_service_name: /plan_point_cluster_grasp
  planning_scene_service_name: /environment_server/set_planning_scene_diff
  arm_inverse_kinematics_plugin: SIA20D_Mesh_manipulator_kinematics/IKFastKinematicsPlugin
segmentation:
  world_frame_id: base_link
  k_nearest: 100
  max_iterations: 400
  sac_method: 6 #ransac = 0
  normal_distance_weight: 0.2
  distance_threshold: 0.0005
  max_radius: 0.028
  min_fitness_score: 0.12
  align_to_top_centroid: true
goal:
  position:
    x: 0
    y: 0
    z: 0
  orientation:
    axis:
      x: 0
      y: 0
      z: 1
    angle: 0
